starter-code/css/main.css

```diff
@@ -6,3 +6,189 @@ Green: #00B172
 White: #FFF
 
 */
+@import url('https://fonts.googleapis.com/css?family=Open+Sans');
+
+*{
```

Comment:
This will be applied to *every* object. I suggest you only apply it to `body`. Two reasons:

- Most elements have default margins and paddings set to 0. Only `body` has some default margin.
- If you set the `font-family` for an element, the effect is going to be propagated to the children. By setting it for `body` you will affect the whole page.